app/models/agents/data_output_agent.rb

```diff
@@ -159,40 +190,54 @@ def receive_web_request(params, method, format)
           interpolated
         end
 
+        now = Time.now
+
         if format =~ /json/
           content = {
             'title' => feed_title,
             'description' => feed_description,
-            'pubDate' => Time.now,
+            'pubDate' => now,
             'items' => simplify_item_for_json(items)
           }
 
           return [content, 200]
         else
-          content = Utils.unindent(<<-XML)
-            <?xml version="1.0" encoding="UTF-8" ?>
-            <rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
-            <channel>
-             <atom:link href=#{feed_url(secret: params['secret'], format: :xml).encode(xml: :attr)} rel="self" type="application/rss+xml" />
-             <atom:icon>#{feed_icon.encode(xml: :text)}</atom:icon>
-             <title>#{feed_title.encode(xml: :text)}</title>
-             <description>#{feed_description.encode(xml: :text)}</description>
-             <link>#{feed_link.encode(xml: :text)}</link>
-             <lastBuildDate>#{Time.now.rfc2822.to_s.encode(xml: :text)}</lastBuildDate>
-             <pubDate>#{Time.now.rfc2822.to_s.encode(xml: :text)}</pubDate>
-             <ttl>#{feed_ttl}</ttl>
-
+          hub_links = push_hubs.map { |hub|
+            <<-XML
+ <atom:link rel="hub" href="#{hub.encode(xml: :attr)}"/>
```

Comment:
I noticed that here you have it wrapped in quotes (`"`), whereas below we have `href=#{feed_url(secret: params['secret'], format: :xml).encode(xml: :attr)} rel="self"`.  Which one of these is right?